diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3ec5481..815bcdf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,10 +16,10 @@ if("${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_BINARY_DIR}")
                     "Please use the Build folder or create your own.")
 endif()
 
-project(svt-av1 VERSION 2.2.0
+project(svt-av1 VERSION 2.2.1
     LANGUAGES C CXX)
 
-set(VERSION_TAG "rc2")
+set(VERSION_TAG "")
 
 if(POLICY CMP0063)
   cmake_policy(SET CMP0063 NEW)
